pkgs/by-name/ng/ngn-k/package.nix

```
@@ -3,44 +3,35 @@
 , stdenvNoLibs
 , fetchFromGitea
 , runtimeShell
-, doCheck ? withLibc && stdenv.hostPlatform == stdenv.buildPlatform
-, withLibc ? true
+, doCheck ? stdenv.hostPlatform == stdenv.buildPlatform
```

```suggestion
, doCheck ? stdenv.buildPlatform.canExecute stdenv.hostPlatform
```
This will help run checks when cross compiling on targets that can actually execute code from the platform they're building for (assuming that is the requirement of running checks for this projects, but correct me if I'm wrong)

---

Useless code now anyways, removed.